content/intro/pricing/rates.md

```diff
@@ -0,0 +1,94 @@
+---
+menu:
+  main:
+    parent: pricing
+title: Rates
+weight: 10
+---
+
+cloud.gov fees are broken into three components:
+
+- Access package
+- Usage
+- Managed services
+
+[Find out more about this pricing model.]({{< relref "intro/pricing/pricing-model.md" >}})
```

Comment:
So we are basically moving "What's included" 2 levels out? I think we should have something here.

Also, if it takes 3 pages to explain how we charge I think there is something broken.